Closed and reopened this PR in an attempt to resolve the following Travis build error:

> AssertionError: Test suit does not fail on the initial contents

--- a/curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/check-radio-buttons-and-checkboxes-by-default.english.md
+++ b/curriculum/challenges/english/01-responsive-web-design/basic-html-and-html5/check-radio-buttons-and-checkboxes-by-default.english.md
@@ -76,10 +76,6 @@ tests:
 ## Solution
 <section id='solution'>
 
-<section id='challengeSeed'>
-
-<div id='html-seed'>
-
 ```html
 <h2>CatPhotoApp</h2>
 <main>
@@ -111,13 +107,4 @@ tests:
 </main>
 ```
 
-</div>
-
-
-
-</section>
-
-```js
-// solution required
-```
 </section>

